### PR TITLE
update title according to design

### PR DESCRIPTION
**Pull request by MagdaN**
- hardcode title
- remove px from scss and use spacer

### CODE REVIEW

File: liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html [hunk 1]
```diff
@@ -9,7 +9,8 @@
                 {% image page.image width-770 as image %}
                 <img src="{{ image.url }}" alt="{{ image.alt }}" class="homepage-header__image" />
                 <div class="homepage-header__text">
-                    <h1 class="cms-title">{{ page.title }}</h1>
+                    <h1 class="homepage-title homepage-title--large">Beteiligung.in/<span class="marked cms-title--normal">jederstadt</span></h1>
```
Comment (fuzzylogic2000) [inline liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html]: Could we just add a class to the <br> and give that display: none for displays bigger than breakpoint?

File: liqd_product/assets/scss/components/_cms.scss [hunk 2]
```diff
@@ -13,12 +12,30 @@
     font-weight: 600;
```
Comment (fuzzylogic2000) [inline liqd_product/assets/scss/components/_cms.scss]: I know, this was here before, but we should probably define a variable for that? demibold?

File: liqd_product/assets/scss/components/_cms.scss [hunk 3]
```diff
@@ -13,12 +12,30 @@
...
 }
 
+.homepage-title {
+    display: inline-block;
+    border-bottom: solid 8px $platform-brand-primary;
+    padding-bottom: 23px;
+}
+
+.homepage-title--large {
+    @media screen and (max-width: $breakpoint) {
```
Comment (fuzzylogic2000) [inline liqd_product/assets/scss/components/_cms.scss]: With the reorganising the heading we would also not ask for min and max and use that one pixel inbetween?

File: liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html [hunk 4]
```diff
@@ -9,7 +9,8 @@
...
+                    <h1 class="homepage-title homepage-title--small">Beteiligung.in/<br><span class="marked cms-title--normal">jederstadt</span></h1>
```
Comment (kleingeist) [inline liqd_product/apps/cms/pages/templates/liqd_product_cms_pages/home_page.html]: would it be possible to use a pseudo `:after` selector to add the break on small screens?

File: liqd_product/assets/scss/components/_cms.scss (hunk 3 above)
Comment (kleingeist) [reply liqd_product/assets/scss/components/_cms.scss]: what do you mean by pixel in between?

**Comment by kleingeist** [comment]
merged with the creation of an issue to rewrite the css and to prevent from duplicating html elements

**Pull request merged**